src/actors/symbolication.rs

```diff
@@ -137,17 +137,17 @@ impl SymbolicationActor {
         symcaches: SymCacheActor,
         cficaches: CfiCacheActor,
         threadpool: ThreadPool,
-    ) -> Self {
+    ) -> Result<Self, procspawn::SpawnError> {
```

Comment:
Might be nice to wrap this in our own error here already so that the caller no longer has to deal with `procspawn`. It seems like an implementation detail of `SymbolicationActor`.